cpp/include/Ice/StreamHelpers.h

```diff
@@ -10,7 +10,11 @@
 #include <iterator>
 #include <ostream>
 
-#if __has_include(<span>)
+#if defined(_MSC_VER) // workaround for MSVC bug
```

Comment:
Is there an issue to link to or can you add a description or what this bug is.

Reply:
I could not find a bug report.

Reply:
This is actually a permissible behavior according to cppreference:
https://en.cppreference.com/w/cpp/preprocessor/include

> A __has_include result of 1 only means that a header or source file with the specified name exists. It does not mean that the header or source file, when included, would not cause an error or would contain anything useful. For example, on a C++ implementation that supports both C++14 and C++17 modes (and provides __has_include in its C++14 mode as a conforming extension), __has_include(<optional>) may be 1 in C++14 mode, but actually #include <optional> may cause an error.

Updated comment accordingly.